decidim-assemblies/app/forms/decidim/assemblies/admin/assemblies_setting_form.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    module Admin
+      # A form object used to create assembly setting from the admin dashboard.
+      class AssembliesSettingForm < Form
+        include TranslatableAttributes
```

Comment:
Not sure if `TranslatableAttributes` is needed as the attribute is `Boolean`, can you check it?